### PR TITLE
Feature/BBL-445 | Identities, Network, Organizations, Workflows and Diagrams 

### PR DESCRIPTION
**Pull request by exequielrafaela**
### Commits on Dec 04, 2020
- @exequielrafaela - BBL-445 | upgrading Makefile live to use the lastest stable ver including latest stable material-mkdocs image - b3008e5

### Commits on Dec 06, 2020
- @exequielrafaela - BBL-445 | adding and updating figures and diagrams - 342f1fa
- @exequielrafaela - BBL-445 | how it works content and format updated - sections: cdn, ci-cd, code-library, compute and costs - 7f92291
- @exequielrafaela - BBL-445 | how-it-work sections updated and added - identities, monitoring and network - a88a1ad
- @exequielrafaela - BBL-445 | how-it-works multiple sections updated: organization, reliability, security and storage - a81384e
- @exequielrafaela - BBL-445 | user-guide base-configuration + base-workflow improved - 63fb9ac
- @exequielrafaela
BBL-445 | user-guide multiple sections updated - compute, identities, network and organization - d650020
- @exequielrafaela - BBL-445 | mkdocs.yml updated with newest page tree structure - 8ef989c

### Commits on Dec 07, 2020
- @exequielrafaela - BBL-445 | updating home illustration icons - ddee7cb

### CODE REVIEW

File: docs/how-it-works/identities/roles.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+<figcaption style="font-size:15px">
+<b>Figure:</b> Assuming an AWS IAM role.
+(Source: Gruntwork.io, 
+<a href="https://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/#iam-roles">
+"How to configure a production-grade AWS account structure using Gruntwork AWS Landing Zone"</a>,
+Gruntwork.io Production deployment guides, accessed November 17th 2020).
+</figcaption>
+
+!!! example "Basic AssumoRole workflow"
```
Comment (diego-ojeda-binbash) [inline docs/how-it-works/identities/roles.md:107]: Small typo: "AssumoRole"  😉

File: docs/user-guide/base-workflow/repo-le-tf-infra-aws-tf-state.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        ```
+          # backend "s3" {
+          #   key = "root/tf-backend/terraform.tfstate"
+          # }
+        ```
+        - Run `make init` and type `yes` when Terraform asks if you want to import the state to the S3 backend
+        - Done. You can remove `terraform.tfstate` now (and also `terraform.tfstate.backup` if available)
+
+## Expected workflow after set up 
+[![asciicast](https://asciinema.org/a/377220.svg)](https://asciinema.org/a/377220)
```
Comment (diego-ojeda-binbash) [inline docs/user-guide/base-workflow/repo-le-tf-infra-aws-tf-state.md:63]: 🙌 🙌 🚀

**Review by diego-ojeda-binbash — APPROVED** [review]

File: docs/how-it-works/identities/roles.md (hunk 1 above)
Comment (exequielrafaela) [reply docs/how-it-works/identities/roles.md:107]: @diego-ojeda-binbash OJ thanks! I'll correct it before merging ✅

Comment (exequielrafaela) [reply docs/how-it-works/identities/roles.md]: @exequielrafaela - BBL-445 | fixing typo - d04310e

**Pull request merged**